pac4j-saml/src/main/java/org/pac4j/saml/context/SAML2ContextProvider.java

```diff
@@ -0,0 +1,169 @@
+/*
+  Copyright 2012 -2014 Michael Remond
```

Comment:
The license header should be `2012 - 2015 pac4j organization` and the author should be kept by the `@author` annotation in the class Javadoc.